bitnami/metallb/templates/daemonset.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "metallb.fullname" . }}-speaker
+  labels: {{- include "metallb.labels" . | nindent 4 }}
+    component: speaker
+spec:
+  selector:
+    matchLabels: {{- include "metallb.matchLabels" . | nindent 6 }}
+      component: speaker
+  template:
+    metadata:
+      labels: {{- include "metallb.labels" . | nindent 8 }}
+        component: speaker
+      {{- if .Values.controller.podAnnotations }}
+      annotations: {{- toYaml .Values.controller.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec:
+{{- include "metallb.speaker.imagePullSecrets" . | nindent 6 }}
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      serviceAccountName: {{ include "metallb.speakerServiceAccountName" . }}
+      terminationGracePeriodSeconds: 0
+      hostNetwork: true
+      containers:
+      - name: speaker
+        image: {{ include "metallb.speaker.image" . }}
+        imagePullPolicy: {{ .Values.speaker.image.pullPolicy }}
+        args:
+        - --port=7472
```

Comment:
Could the port be configurable? In that case I would use a template here instead of hardcode the value, this is also in the line `45` for the `containerPort`. So in case you need to change it, it needs to be changed in once.
This also applies to the controller.

Reply:
I can test that.